tests/unit/backend/recipe/algorithm/test_FitMultiplePeaksAlgorithm.py

```diff
@@ -37,39 +43,39 @@ def test_execute():
             DataY=[1] * 6,
             NSpec=6,
         )
-        fitIngredients = Ingredients.parse_file(Resource.getPath("inputs/predict_peaks/input_good_ingredients.json"))
+        peaks = SculleryBoy().prepDetectorPeaks({"good": ""})
         fmpAlgo = FitMultiplePeaksAlgorithm()
         fmpAlgo.initialize()
         fmpAlgo.setPropertyValue("InputWorkspace", wsName)
-        fmpAlgo.setProperty("DetectorPeakIngredients", fitIngredients.json())
+        fmpAlgo.setProperty("DetectorPeaks", list_to_raw(peaks))
         fmpAlgo.execute()
         wsGroupName = fmpAlgo.getProperty("OutputWorkspaceGroup").value
         assert wsGroupName == "fitPeaksWSGroup"
         wsGroup = list(mtd[wsGroupName].getNames())
         expected = [
-            "testWS_fitted_peakpositions_0",
-            "testWS_fitted_params_0",
-            "testWS_fitted_0",
-            "testWS_fitted_params_err_0",
-            "testWS_fitted_peakpositions_1",
-            "testWS_fitted_params_1",
-            "testWS_fitted_1",
-            "testWS_fitted_params_err_1",
-            "testWS_fitted_peakpositions_2",
-            "testWS_fitted_params_2",
-            "testWS_fitted_2",
-            "testWS_fitted_params_err_2",
-            "testWS_fitted_peakpositions_3",
-            "testWS_fitted_params_3",
-            "testWS_fitted_3",
-            "testWS_fitted_params_err_3",
-            "testWS_fitted_peakpositions_4",
-            "testWS_fitted_params_4",
-            "testWS_fitted_4",
-            "testWS_fitted_params_err_4",
-            "testWS_fitted_peakpositions_5",
-            "testWS_fitted_params_5",
-            "testWS_fitted_5",
-            "testWS_fitted_params_err_5",
+            "fitPeaksWSGroup_fitted_peakpositions_0",
+            "fitPeaksWSGroup_fitted_params_0",
+            "fitPeaksWSGroup_fitted_0",
+            "fitPeaksWSGroup_fitted_params_err_0",
+            "fitPeaksWSGroup_fitted_peakpositions_1",
+            "fitPeaksWSGroup_fitted_params_1",
+            "fitPeaksWSGroup_fitted_1",
+            "fitPeaksWSGroup_fitted_params_err_1",
+            "fitPeaksWSGroup_fitted_peakpositions_2",
+            "fitPeaksWSGroup_fitted_params_2",
+            "fitPeaksWSGroup_fitted_2",
+            "fitPeaksWSGroup_fitted_params_err_2",
+            "fitPeaksWSGroup_fitted_peakpositions_3",
+            "fitPeaksWSGroup_fitted_params_3",
+            "fitPeaksWSGroup_fitted_3",
+            "fitPeaksWSGroup_fitted_params_err_3",
+            "fitPeaksWSGroup_fitted_peakpositions_4",
+            "fitPeaksWSGroup_fitted_params_4",
+            "fitPeaksWSGroup_fitted_4",
+            "fitPeaksWSGroup_fitted_params_err_4",
+            "fitPeaksWSGroup_fitted_peakpositions_5",
+            "fitPeaksWSGroup_fitted_params_5",
+            "fitPeaksWSGroup_fitted_5",
```

Comment:
Note: these changed because FitMultiplePeaksAlgorithm now bases the output grouped workspace names on the output, not input